### PR TITLE
Address #5902 add 1-liner description to CVEs

### PR DESCRIPTION
**Pull request by fpesce**

### CODE REVIEW

**Comment by geeknoid** [comment]
You can take out the version information from the descriptions. It's already captured in the front-matter as explicit metadata, and it's rendered on the bulletin summary page.

File: content/en/news/security/istio-security-2019-001/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ISTIO-SECURITY-2019-001
 subtitle: Security Bulletin
-description: Security vulnerability disclosure for CVE-2019-12243.
+description: Incorrect access control affecting Istio 1.1 to 1.1.6.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2019-001/index.md]:
```suggestion
description: Incorrect access control.
```

File: content/en/news/security/istio-security-2019-002/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ISTIO-SECURITY-2019-002
 subtitle: Security Bulletin
-description: Security vulnerability disclosure for CVE-2019-12995.
+description: Denial of service affecting JWT access token parsing in versions 1.0 to 1.0.8, 1.1 to 1.1.9, and 1.2 to 1.2.1 of Istio.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2019-002/index.md]:
```suggestion
description: Denial of service affecting JWT access token parsing.
```

File: content/en/news/security/istio-security-2019-003/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ISTIO-SECURITY-2019-003
 subtitle: Security Bulletin
-description: Security vulnerability disclosure for CVE-2019-14993.
+description: Denial of service in regular expression parsing. This affects Istio versions 1.1 to 1.1.12 and 1.2 to 1.2.3.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2019-003/index.md]:
```suggestion
description: Denial of service in regular expression parsing.
```

File: content/en/news/security/istio-security-2019-004/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ISTIO-SECURITY-2019-004
 subtitle: Security Bulletin
-description: Security vulnerability disclosure for multiple CVEs.
+description: Multiple denial of service vulnerabilities related to HTTP2 support in Envoy. This affects Istio versions 1.1 to 1.1.12 and 1.2 to 1.2.3.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2019-004/index.md]:
```suggestion
description: Multiple denial of service vulnerabilities related to HTTP2 support in Envoy.
```

File: content/en/news/security/istio-security-2019-005/index.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ISTIO-SECURITY-2019-005
 subtitle: Security Bulletin
-description: Security vulnerability disclosure for CVE-2019-15226.
+description: Denial of service caused by the presence of numerous HTTP headers. This affects Istio 1.1 to 1.1.15, 1.2 to 1.2.6 and 1.3 to 1.3.1.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2019-005/index.md]:
```suggestion
description: Denial of service caused by the presence of numerous HTTP headers in client requests.
```

File: content/en/news/security/istio-security-2019-006/index.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ISTIO-SECURITY-2019-006
 subtitle: Security Bulletin
-description: Security vulnerability disclosure for CVE-2019-18817.
+description: Denial of service affecting Istio 1.3 to 1.3.4.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2019-006/index.md]:
```suggestion
description: Denial of service.
```

**Pull request merged**